portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,15 @@
+package com.google.sps.data;
+import java.util.Date;
+
+public final class Comments {
```

Comment:
This class looks to represent a single comment entity, not a plurality--please rename to `Comment`).